app/deepseg.cc

```diff
@@ -268,15 +326,15 @@ int main(int argc, char* argv[]) try {
 	size_t threads= 2;
 	size_t width  = 640;
 	size_t height = 480;
-	const char *back = nullptr; // "images/background.png";
-	const char *vcam = "/dev/video0";
-	const char *ccam = "/dev/video1";
+	const char *back = nullptr;
+	const char *vcam = "/dev/video1";
+	const char *ccam = "/dev/video0";
 	bool flipHorizontal = false;
 	bool flipVertical   = false;
 	int fourcc = 0;
 	size_t blur_strength = 0;
 
-	const char* modelname = "models/selfiesegmentation_mlkit-256x256-2021_01_19-v1215.f16.tflite";
+	const char* modelname = "selfiesegmentation_mlkit-256x256-2021_01_19-v1215.f16.tflite";
```

Comment:
Default is now the installed model - developers need to supply a path or install a build... good or bad?